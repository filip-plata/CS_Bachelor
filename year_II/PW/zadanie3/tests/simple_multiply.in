5 1 2
1 x[0] = ((-x[1]) * (x[1] + (-x[1])))
2 x[1] -1000
3 x[1] -966
4 x[1] 1000
5 x[1] 850
