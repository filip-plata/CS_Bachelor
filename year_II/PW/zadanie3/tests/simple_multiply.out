1 P
2 P 0
3 P 0
4 P 0
5 P 0
